apps/web-giddh/src/app/ledger/components/new-ledger-entry-panel/new-ledger-entry-panel.component.html

```diff
@@ -499,28 +480,18 @@
                                             blankLedger.voucherType === 'pay' ||
                                             blankLedger.voucherType === 'rcpt' ||
                                             blankLedger.voucherType === 'advance-receipt' ||
-                                            (voucherApiVersion === 2 && blankLedger.voucherType == 'jr')
+                                            blankLedger.voucherType == 'jr'
                                         "
                                     >
                                         <div class="form-group">
-                                            <label class="default-color">{{
-                                                voucherApiVersion === 2
```

Comment:
please check in over file for voucher version condition please do not remove this its shows in 2 only